app/controllers/posts_controller.rb

```diff
@@ -466,6 +466,22 @@ def hide_revision
     render body: nil
   end
 
+  def purge_revisions
+    guardian.ensure_can_purge_post_revisions!
```

Comment:
Since this is a destructive operation, do we want to add a (sane) rate limit to ensure a rogue staff member doesn't go wild and batch purge revisions? 🤔

Reply:
[Added](https://github.com/discourse/discourse/pull/19913/files#diff-41547fca0ee5b1e4a7a2be25356b71d2fc7389aa1e86fe97fc10cd6f48d1c6dbR476-R482) rate limiter, and set limits to 20 posts a minute.